evansmusomi/burger-builder
# Adds Logout Ability

*What this does*
- Updates nav items (ui) based on auth state
- Adds logout ability

*How to verify*
- A user can see the Authenticate menu when not logged in
- A user can see the Logout menu when logged in
- A user can log out of the app and be redirected to the main screen when they select the logout nav menu
